sfn-query-large-datasets-cdk/README.md

```diff
@@ -0,0 +1,110 @@
+| Glue Crawler     | GlueCrawler        |  
+  
+## Start an Athena query
+
+This workflow demonstrates how to ingest a large data set in Amazon S3 and partition it through AWS Glue Crawlers, then execute Amazon Athena queries against that partition. Deploying this sample project creates an AWS Step Functions state machine, an Amazon S3 Bucket, an AWS Glue crawler, and an Amazon SNS topic.
+
+In this project, the Step Functions state machine invokes an AWS Glue crawler that partitions a large dataset in Amazon S3. Once the AWS Glue crawler returns a success message, the workflow executes Athena queries against that partition. Once query execution is successfully complete, an Amazon SNS notification is sent to an Amazon SNS topic.
+
+* An Amazon Athena query
+* An AWS Glue crawler
+* AWS Lambda Functions
+* An Amazon Simple Notification Service topic
+* Related AWS Identity and Access Management (IAM) roles
+
+Learn more about this workflow at Step Functions workflows collection: https://docs.aws.amazon.com/step-functions/latest/dg/sample-query-large-datasets.html
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. If this is your first time using AWS CDK, bootstrap your [environment](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_bootstrap).
+    ```
+    cdk bootstrap aws://{your-aws-account-number}/{your-aws-region}
+    ```
+
+2. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/step-functions-workflows-collection
+    ```
+3. Change directory to the pattern directory:
+    ```
+    cd sfn-query-large-datasets-cdk
+    ```
+4. Create a Python virtual environment and install the requirements:
+    ```
+    python3 -m venv .venv
+    source .venv/bin/activate
+    python3 -m pip install -r requirements.txt
+    ```
+5. From the command line, use CDK to deploy the AWS resources for the workflow as specified in the ```app.py``` file:
+    ```
+    cdk deploy
+    ```
+6. During the prompts:
```

Comment:
Shall we add more instructions for the cdk deploy prompts? Like "accept default values but when asked to deploy say yes" or whatever it applies.